RESIZE BUTTON FUNCTION AND DELETE BUTTON WITH LETTER "D"

leonardopalma opened the pull request:
# Purpose / Goal
Please specify here what you're planning to achieve by this pull request and how it can help in regard of this work-space. 

Please mention the issue number, if exists.

# Type
Please mention the type of PR


* [ ] Bug Fix
* [ ] Refactoring / Technology upgrade
* [x] New Feature
* [ ] Documentation
* [ ] Other : | Please Specify |


**Note** : Please ensure that you've read contribution [guidelines](CONTRIBUTING.md) before raising this PR.

amitguptagwl commented:
Thanks for the PR. It'll be good if you embed some GIF for the visualization. Please also ensure that you've tested the application for rest of the features as we don't have unit tests at this moment.

leonardopalma commented:
i implemented in this new file the button resize that consist in enlarge or not shapes area, i implemented to button delete for mac user : Press letter " d " ; It would be nice if he implemented it in your project;   I'm waiting for your news, thanks. 
[ResizeZip.zip](https://github.com/NaturalIntelligence/imglab/files/3236906/ResizeZip.zip)

amitguptagwl commented:
why do you wanna assign `d` to delete?
